common/models/wordbreakers/test/test-dict-breaker.js

```diff
@@ -159,5 +180,44 @@ describe('dictionary-based wordbreaker', () => {
         assertSpanSplit(text, actualSplit, expectedSplit);
       });
     });
+
+    describe('breaking with sil.km.gcc 1.0', () => {
+      const fixture = jsonFixture('tries/sil.km.gcc - 1.0');
+
+      // Uses the original lexical-model's backing data, but swaps in the
+      // dictionary-based wordbreaker.
+      const model = new TrieModel(fixture);
```

Comment:
It would be better to instead directly use the `Trie` type as exposed through #12080, but that's not available here yet.